sdcm/nemesis.py

```diff
@@ -2120,22 +2124,34 @@ def _parse_toppartitions_output(output):
             return toppartitions
 
         def generate_random_parameters_values():  # pylint: disable=invalid-name
-            ks_cf_list = self.cluster.get_non_system_ks_cf_list(self.target_node)
+            ks_cf_list = self.cluster.get_any_ks_cf_list(self.target_node)
+            filter_ks_cf = []
+            ks = ''
+            cf = ''
+
             if not ks_cf_list:
-                raise UnsupportedNemesis('User-defined Keyspace and ColumnFamily are not found.')
+                raise UnsupportedNemesis('Any Keyspace and ColumnFamily are not found.')
```

Comment:
No Keyspace and ColumnFamily were not found.

Reply:
fixed